### PR TITLE
Add old article team standards to code standards

### PR DESCRIPTION
**Pull request by jamesbhobbs**
Updates code standards with standards we've had in place for 18 months.

### CODE REVIEW

File: docs/Code-Standards.md [hunk 1]
```diff
@@ -2,7 +2,11 @@
 
 ## Coding Approach
```
Comment (thekp) [inline docs/Code-Standards.md]: I would remove this subheading, as it feels like unnecessary clutter. Even though I originally put it there 😅

**Review by thekp — CHANGES_REQUESTED** [review]

File: docs/Code-Standards.md (hunk 1 above)
Comment (jamesbhobbs) [reply docs/Code-Standards.md]: lol, ok

**Review by pjlee11 — APPROVED** [review]
Looks good

**Review by thekp — APPROVED** [review]

**Comment by jamesbhobbs** [comment]
This doesn't need testing.

**Pull request merged**